packages/manager/core/generator/app/templates/src/components/Breadcrumb/Breadcrumb.tsx.hbs

```diff
@@ -1,67 +1,21 @@
-/**
- *  TODO - Move in the future
- *  Super components manager library
- */
-
-import React, { useEffect, useState } from 'react';
-import { Params, useMatches } from 'react-router-dom';
+import React from 'react';
 import { OsdsBreadcrumb } from '@ovhcloud/ods-components/breadcrumb/react';
-
-export type BreadcrumbHandleParams = {
-  data: unknown;
-  params: Params<string>;
-};
-
-type BreadcrumbItem = {
-  label: string;
-  href: string;
-};
-
-type MatchHandle = {
-  breadcrumb?: ({ data, params }: BreadcrumbHandleParams) => string;
-};
-
-type Match = {
-  handle: MatchHandle;
-  data: unknown;
-  params: Params<string>;
-};
-
-function Breadcrumb(): JSX.Element {
-  const matches = useMatches();
-  const [categoryCrumbs, setCategoryCrumbs] = useState<BreadcrumbItem[]>([]);
-  const [matchCrumbs, setMatchCrumbs] = useState<BreadcrumbItem[]>([]);
-
-  useEffect(() => {
-    const items = matches.map(async (match) => {
-      const { handle, data, params }: Match = match;
-      const breadcrumb = await handle?.breadcrumb;
-      const crumb = {
-        label: await breadcrumb?.({ data, params }),
-        href: match.pathname,
-      };
-      return crumb;
-    });
-
-    Promise.all(items)
-      .then((breadcrumbs) => {
-        const validCrumbs = breadcrumbs.filter(
-          ({ label, href }) => !!label && !!href,
-        );
-        setMatchCrumbs(validCrumbs);
-      })
-      .catch((error) => {
-        console.error('Error fetching breadcrumbs:', error);
-      });
-  }, [matches]);
-
-  const rootName = `${window.location.origin}/#/{{appName}}`;
-  const crumbs = [...categoryCrumbs, ...matchCrumbs];
-  const data = crumbs.map((crumb, index) => ({
-    label: crumb.label,
-    href: `${rootName}${crumb.href}`,
-  }));
-  return <OsdsBreadcrumb items={data} />;
+import {
+  useBreadcrumb,
+  BreadcrumbProps,
+} from '@ovh-ux/manager-react-core-application';
+
+function Breadcrumb({ rootLabel }: BreadcrumbProps): JSX.Element {
+  const breadcrumbItems = useBreadcrumb({ rootLabel });
+
+  return (
+    <OsdsBreadcrumb
+      items={breadcrumbItems.map((item) => ({
+        label: item.label,
+        href: item.href,
+      }))}
```

Comment:
Do we really need this mapping ?

Reply:
But breadcrumbItem is already an array and items already has props label and href. So the only thing interesting here is to filter other properties but I don't think there are other properties that could impact the OsdsBreadcrumb